app/controllers/placements/providers/partner_schools/placements_controller.rb

```diff
@@ -0,0 +1,26 @@
+class Placements::Providers::PartnerSchools::PlacementsController < ApplicationController
+  before_action :set_provider, only: %i[index show]
+  before_action :set_partner_school, only: %i[index show]
+  def index
+    placements_assigned_to_provider = @partner_school.placements.where(provider_id: @provider.id)
+    other_placements = @partner_school.placements.where.not(id: placements_assigned_to_provider.ids)
+    @placements_assigned_to_provider = placements_assigned_to_provider.decorate
+    @other_placements = other_placements.decorate
+  end
+
+  def show
+    @placement = @partner_school.placements.find(params.require(:id)).decorate
+    @partner_school = @partner_school.decorate
+  end
+
+  private
+
+  def set_partner_school
+    @partner_school = @provider.partner_schools.find(params.require(:partner_school_id)).becomes(Placements::School)
+  end
+
+  def set_provider
+    provider_id = params.fetch(:provider_id)
+    @provider = current_user.providers.find(provider_id)
+  end
+end
```

Comment:
You might prefer to define a 'placements scope' private method for this controller, to be used by both `#index` and `#show`.

```ruby
def placements_scope
  @partner_school.placements
end
```

and then

```ruby
def index
  @placements_assigned_to_provider = placements_scope.where(provider: @provider).decorate
  @other_placements = placements_scope.where.not(provider: @provider).decorate
end

def show
  @placement = placements_scope.find(params.require(:id)).decorate
  # @partner_school = 
end
```